downloads/swan-lake-release-notes/swan-lake-beta3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,780 @@
+type Person record {
+};
+
+configurable Person person = ?;
+```
+
+the values can be provided in the `Config.toml` file as follows.
+
+
+```toml
+[person]
+intVal = 22
+floatVal = 22.33
+stringVal = "abc"
+arrVal = [1,2,3]
+mapVal.a = "a"
+mapVal.b = 123
+```
+
+The additional fields that are created from the TOML values will have the following types.
+
+* TOML Integer - `int`
+* TOML Float - `float`
+* TOML String - `string`
+* TOML Boolean - `boolean`
+* TOML Table - `map<anydata>`
+* TOML Table array - `map<anydata>[]`
+
+Similarly, if a configurable variable with a record type that contains a rest field is defined in the following way,
+
+```ballerina
+public type Numbers record {|
+   int ...;
+|};
+
+configurable Numbers numbers = ?;
+```
+
+the values can be provided in the `Config.toml` file as follows.
+
+
+```toml
+num1 = 11
+num2 = 26
+```
+
+#### Improved the Printed Error Stacktrace to Include the Cause
+
+The error stack trace has been improved to include the error cause locations. Stack frames of the wrapped error causes are also added to the stack trace.
+
+For the following example,
+
+```ballerina
+public function main() {
+    panic bar();
+}
+function bar() returns error {
+    return error("a", y());
+}
+function y() returns error {
+    return x();
+}
+function x() returns error {
+    return error("b");
+}
+```
+
+the expected stack trace will be as follows.
+
+```
+error: a
+at cause_location.0:bar(main.bal:6)
+cause_location.0:main(main.bal:2)
+cause: b
+at cause_location.0:x(main.bal:14)
+cause_location.0:y(main.bal:10)
+... 2 more
+```
+#### New Runtime Java APIs
+
+##### Invoke the Ballerina Object Method Asynchronously
+
+New Java Runtime APIs are introduced to execute the Ballerina object method from Java. The object method caller can decide whether to execute the object method sequentially or concurrently using the appropriate API.
+
+If the caller can ensure that the given object and object method is isolated and no data race is possible for the mutable state with given arguments, they can use the `invokeMethodAsyncConcurrently` method, or otherwise, the `invokeMethodAsyncSequentially` method.
+
+```java
+boolean isIsolated = object.getType().isIsolated(methodName);
+if (isIsolated) {
+    BFuture bFuture = env.getRuntime().invokeMethodAsyncConcurrently(object, methodName, strandName, metadata, 
+                                    callback, properties, returnType, args);
+} else {
+    BFuture bFuture = env.getRuntime().invokeMethodAsyncSequentially(object, methodName, strandName, metadata,
+                                    callback, properties, returnType, args);
+}
+```
+
+> **Info:** The previous `invokeMethodAsync` methods are deprecated.
+
+#####  API to Retrieve the Isolated Ballerina Object or Object Method 
+
+The two new APIs below are introduced to the `ObjectType`.
+
+```java
+boolean isIsolated();
+boolean isIsolated(String methodName);
+```
+
+#### Removed the Package Version from the Runtime
```

Comment:
Shouldn't the title reflect the fully-qualified part?

Reply:
```suggestion
#### Removed Fully-Qualified Package Version from Runtime
```